### PR TITLE
fix bar chart overflow

### PR DESCRIPTION
**Pull request by RCode-Blue**
## Overview of the Pull Request:
Placed `<HichChartsReact>` inside a `<Container>` so to prevent bar chart overflow
Refactored the bar chart config so that is returns an image depending on the data category.

## link to Trello card or Github issue
n/a

## How Has This Been Tested?

The section should look like the following screenshot:
image

Data comes from local JSON file.

## Pull Request Checklist:

Make sure the following checkboxes`[ ]` are checked with `x` before sending your PR -- thank you!

- [ ] Have you included a link Trello card / Github issue and written sufficient description to help others review your work?
- [x] Is your pull request up-to-date with `main` branch?
- [x] Have you checked That code is well formatted? Did `npx prettier --check .` return no warnings/errors?
- [x] Have you written instructions on how to test that your changes work as expected?
- [x] Have you tested your work thoroughly on your local machine to ensure you are not breaking anything?

### CODE REVIEW

**Comment by OlaShabalina** [comment]
hey @RCode-Blue 

When I run on my machine or click on the [Preview](https://council-emissions-calculator-spike-git-3fee2e-codeforaustralia.vercel.app/results), the graphs are overflowing the containers. We probably need to do something with the container height to fix it.

image

**Comment by RCode-Blue** [comment]
Sorry, I didn't notice that. Will get on it.

File: utils/mathUtils.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const formatCommaSeparators = (number) => {
+  return number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+};
```
Comment (hqtan) [inline utils/mathUtils.js:3]: nice work on this little utility to add comma separators in numbers @RCode-Blue ! 😍

**Review by hqtan — APPROVED** [review]
nice work @RCode-Blue !

I'm happy with this PR!
Will wait for @OlaShabalina to give a final 👍🏽 before merging this.

**Review by OlaShabalina — APPROVED** [review]

**Comment by OlaShabalina** [comment]
Well done @RCode-Blue ! All good from my side @hqtan

**Pull request merged**